core/domain/question_domain.py

```diff
@@ -34,6 +34,7 @@
 # This takes additional 'property_name' and 'new_value' parameters and,
 # optionally, 'old_value'.
 CMD_UPDATE_QUESTION_PROPERTY = 'update_question_property'
+CMD_ADD_QUESTION_FROM_SUGGESTION = 'add_question_from_suggestion'
 
 # The following commands are deprecated, as these functionalities will be
 # handled by a QuestionSkillLink class in the future.
```

Comment:
Questions hasn't been launched yet. So I think we can just delete these altogether? There's no backend stuff to clean up yet.

Reply:
Done